chart/compass/templates/tenant-fetcher-job.yaml

```diff
@@ -242,6 +242,8 @@ kind: ServiceAccount
 metadata:
   name: {{ $.Chart.Name }}-tenant-fetcher-job-{{ $tenantFetcherName }}
   namespace: {{ $.Release.Namespace }}
+  annotations:
+    iam.gke.io/gcp-service-account: "{{ $.Values.global.sqlproxy.proxyUser }}@sap-{{ $.Values.global.sqlproxy.cloudsql.instance.project }}.iam.gserviceaccount.com"
```

Comment:
We should keep the `@sap` part to ourselves and either add it to the `Values.global.sqlproxy.proxyUser` overrides in the config repo, or simplify this, and instead of user and project part we can have just "sqlProxyServiceAccount" override - they are not used separately in other places anyways